src/T2DMap.cpp

```diff
@@ -4535,6 +4535,29 @@ void T2DMap::wheelEvent(QWheelEvent* e)
     e->ignore();
 }
 
+void T2DMap::changeMapperSettings(int roomSize, int lineSize, bool round, bool info, bool ids, bool viewOnly, bool showPanel)
+{
+  if (mpHost) {
+    this->mBubbleMode = round;
+    mpHost->mBubbleMode = this->mBubbleMode;
+    this->mShowInfo = info;
+    mpHost->mShowInfo = this->mShowInfo;
+    this->mShowRoomID = ids;
+    mpHost->mShowRoomID = this->mShowRoomID;
+    mpHost->mShowPanel = showPanel;
+    this->setMapViewOnly(viewOnly);
+    this->setExitSize(lineSize);
+    this->setRoomSize((float)roomSize / 10.0);
```

Comment:
Don't use `C` style cases - use *one* of the `C++` style ones - and the suggested one is:
```suggestion
    this->setRoomSize(static_cast<float>(roomSize / 10.0));
```
Although TBH you don't actually need a cast at all IIRC.